wire-ios-data-model/Tests/MLS/MockMLSService.swift

```diff
@@ -47,12 +47,14 @@ class MockMLSService: MLSServiceInterface {
         var performPendingJoins: [Void] = []
         var wipeGroup = [MLSGroupID]()
         var generateConferenceInfo = [(MLSGroupID, MLSGroupID)]()
-
+        var joinGroup = [MLSGroupID]()
     }
 
     // MARK: - Properties
 
     var calls = Calls()
+    var mockConversations: [ZMConversation] = []
+    var conversationExistsResponses: [MLSGroupID: Bool] = [:]
```

Comment:
I suggested to remove this and to follow the same mocking pattern as the other methods in this class. Take a look at how `FetchAndRepairGroupMock` is defined and used

Reply:
Fixed the test!